daemon/README

```diff
@@ -0,0 +1,193 @@
+Existing User Tracking
+======================
+
+- tracking is done by:
+  - monitoring /etc/passwd file
+
+- maintained data:
+  - minimum UID
+  - maximum UID
+  - set of valid UID values
+
+- UID added:
+  - no action needed
+
+- UID removed:
+  - user must be dropped from settings data [NOT DONE]
+
+User Session Tracking
+=====================
+
+- tracks UID of current user
+
+- UID is used for:
+  - permissions checking/prompting
+
+- UID changed:
+  - cancels pending/queued prompts [NOT DONE]
+
+Permission File Tracking
+========================
+
+- tracking is done by:
+  - monitoring /etc/sailjail/permissions directory
+
+- maintained data:
+  - set of PERMISSION names
+
+- PERMISSION added/removed:
```

Comment:
Yes, it triggers re-evaluation. But that removing granted permission vs. new permission got added glitch mentioned elsewhere is not yet handled.

Reply:
Installed permission files are used for evaluating effective application permissions in desktop files. Those effective permissions are evaluated on startup. And re-evaluated whenever permission files are added / removed. If effective permissions for application change, that triggers settings re-evaluation. With the latest changes: introducing new permission to effective set for application (adding permission file unmasks permission already present in desktop / desktop file is edited so that it has a new permission) -> application allowed state is cleared.